src/main/java/domain/Winners.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+
+    private final List<Car> cars = new ArrayList<>();
+
+    public Winners(List<CarName> carNames) {
+        for (CarName carName : carNames) {
+            cars.add(new Car(carName));
+        }
+    }
+
+    public List<String> getWinners() {
+        List<String> winners = new ArrayList<>();
+
+        for (Car car : cars) {
+            if (isWinner(getMaxPosition(), car.getPosition())) {
+                winners.add(car.getCarName());
+            }
+        }
+
+        return winners.stream().map(String::toString).collect(Collectors.toList());
```

Comment:
```suggestion
        return winners.stream()
        .map(String::toString)
        .collect(Collectors.toList());
```
한줄당 하나의 `.`을 사용해주시면 좀 더 가독성이 좋아질거 같아요 :)

Reply:
한줄당 하나의 .을 사용하도록 수정하였습니다!